.travis.yml

```diff
@@ -55,7 +55,7 @@ after_script:
 
 branches:
   only:
-    - /.*/
+    - master
```

Comment:
This line should be explained in comment what it will actually do.

Reply:
It was a little ambiguous before due to the regular expression, but now its loud and clear as to what is happening here.